.github/workflows/connector-performance-cron.yml

```diff
@@ -1,34 +1,44 @@
 name: Connector Performance Harness Cron
-on: workflow_dispatch
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "30 5,17 * * *"
+  workflow_dispatch: # for manual triggers
 
 jobs:
   postgres-1m-run:
     uses: ./.github/workflows/connector-performance-command.yml
     with:
-      connector: connectors/source-postgres
+      connector: "connectors/source-postgres"
       dataset: 1m
```

Comment:
Are all dataset going to run every time? is this needed?

Reply:
actually that's something I want to discuss with you. When we are going to support and test against full_refresh and incremental, and with additional mongoDB support, we will have maximum 24 combinations between connectors/dataset/sync mode. I feel we do not need to enumerate all datasets - what is the original purpose of defining 3 different datasets for performance testing?

Reply:
I don't think 1m is giving us something that's very different than 10 or 20m.
I'm ok with leaving other datasets for manual testing where you may want to check a more specialized cases.

Reply:
sounds good. removed 10m/20m for both connectors.